worker/categorize.py

```diff
@@ -181,11 +184,23 @@ def _memoize_article_links(self, ngrams):
     def _scrape_wiki_links(self, titles):
         unscraped_titles = filter(lambda x: x not in self._memoized_article_links, titles)
 
-        for i in xrange(0, len(unscraped_titles), 5):
+        current_title_index = 0
 
-            grouped_titles = unscraped_titles[i:i + 5]
+        while current_title_index < len(unscraped_titles):
+
+            title_grouping_size = DEFAULT_ARTICLES_PER_WIKI_CALL
+
+            titles_length = sum(len(s) for s in unscraped_titles[current_title_index:current_title_index + title_grouping_size])
+
+            if MAX_URL_LENGTH - len(WIKIPEDIA_LINKS) \
+               - titles_length - title_grouping_size > 0:
```

Comment:
```MAX_URL_LENGTH > len(WIKIPEDIA_LINKS) + titles_length + title_grouping size``` is the good condition , so shouldn't that mean that it should be ```<``` if we're going to switch ```title_grouping_size```?